### PR TITLE
Added iOS Audio Classifier Swift and ObjC Tests

### PR DESCRIPTION
**Pull request by priankakariatyml**
1. Added swift tests
2. Added more elaborate objective c tests
3. Couple of API updates to make interfaces more friendly for swift
4. Removed unwanted dependencies in build file

### CODE REVIEW

File: tensorflow_lite_support/ios/task/audio/core/BUILD [hunk 1]
```diff
@@ -12,7 +12,6 @@ objc_library(
         "sources/TFLFloatBuffer.h",
     ],
     module_name = "TFLFloatBuffer",
-    deps = ["//third_party/apple_frameworks:Foundation"],
```
Comment (priankakariatyml) [inline tensorflow_lite_support/ios/task/audio/core/BUILD:15]: @khanhlvg @schmidt-sebastian These dependencies get added every time a PR is merged.
These are not needed. Their inclusion leads to Bazel build failure in my local.

Comment (schmidt-sebastian) [reply tensorflow_lite_support/ios/task/audio/core/BUILD:15]: Will update our script that does the copy to GitHub. Thanks for the heads up.

**Comment by khanhlvg** [comment]
Merged in https://github.com/tensorflow/tflite-support/commit/426885e8e65d9ee7c9a88172f0826d6d478b1495

File: tensorflow_lite_support/ios/test/task/audio/audio_classifier/TFLAudioClassifierTests.m [hunk 2]
```diff
@@ -199,18 +202,18 @@ - (void)validateCategoriesForInferenceWithAudioRecord:(NSArray<TFLCategory *> *)
...
-                 0.003906,                 // expectedScore
-                 @"Narration, monologue",  // expectedLabel
-                 nil                       // expectedDisplaName
+}
+
+- (void)validateCategoriesForInferenceWithLabelDenyList:(NSArray<TFLCategory *> *)categories {
+  VerifyCategory(categories[0],
+                 500,                    // expectedIndex
+                 0.019531,               // expectedScore
+                 @"Inside, small room",  // expectedLabel
```
[the remark this answers is not in the document]
Comment (khanhlvg) [reply tensorflow_lite_support/ios/test/task/audio/audio_classifier/TFLAudioClassifierTests.m:215]: Pinged you on Chat for this.

**Review by khanhlvg — APPROVED** [review]

**Review by schmidt-sebastian — APPROVED** [review]

**Comment by khanhlvg** [comment]
Merged in https://github.com/tensorflow/tflite-support/commit/9023f4074b248ccd5a593c90fbb7778f5af92f87

**Pull request closed without merging**